build_tools/cmake/iree_copts.cmake

```diff
@@ -113,6 +113,7 @@ set(LLVM_INCLUDE_TESTS OFF)
 set(LLVM_INCLUDE_BENCHMARKS OFF)
 set(LLVM_APPEND_VC_REV OFF)
 set(LLVM_ENABLE_IDE ON)
+set(LLVM_ENABLE_RTTI ON)
```

Comment:
This line I'm not sure about (and wanted @stellaraccident 's feedback on), since the Google style guide in general disallows RTTI: https://google.github.io/styleguide/cppguide.html#Run-Time_Type_Information__RTTI_ . If we need it within LLVM (which the issue comments pointed out) then I guess it's fine?

Reply:
I'm fine with it for now -- that part of the style guide must bow to pragmatism, especially with respect to third party integrations.

With that said, it isn't clear to me why this is needed (based on the errors, I believe that it is, but I think it needs more root cause analysis). Can you please create an issue specifically for debugging this so we don't lose track? It will be substantially easier to root-cause once the build system is generally working, so we should proceed.

Reply:
I opened #202 to follow up.